classes/Hook.php

```diff
@@ -129,10 +129,17 @@ class HookCore extends ObjectModel
     public function add($autodate = true, $null_values = false)
     {
         Cache::clean('hook_idsbyname');
+        Cache::clean('hook_active');
 
         return parent::add($autodate, $null_values);
     }
 
+    public function clearCache($all = false)
+    {
+        Cache::clean('hook_*');
+        parent::clearCache($all); // TODO: Change the autogenerated stub
```

Comment:
Why the comment, it's perfectly fine to cal the parent function it's even recommended

Reply:
Oh that's PHPStorm that autogenerated the method, and I forgot to change it.